### ISSUE TITLE
Minor error "minVersion" specification.

### ISSUE THREAD

**Issue by GuldTasken** [qouteall/ImmersivePortalsModForForge#51]
21:37:03.735
Mixin config assets/immersive_portals/immersive_portals.mixins.json does not specify "minVersion" property
21:37:03.739
Mixin config assets/immersive_portals/immersive_portals.mixins_ma.json does not specify "minVersion" property
21:37:03.751
Mixin config assets/immersive_portals/immersive_portals.mixins_client.json does not specify "minVersion" property
21:37:03.762
Mixin config assets/immersive_portals/immersive_portals.mixins_ma_client.json does not specify "minVersion" propert

 
↑ Got this minor error in the 1.15.2 version. ↑

Also, doesn't seem to work for me, in my modpack(1.15.2). Not sure if it may conflict with any of these nether related addons;
https://www.curseforge.com/minecraft/mc-mods/nether-portal-spread
https://www.curseforge.com/minecraft/mc-mods/netherportalfix
https://www.curseforge.com/minecraft/mc-mods/smaller-nether-portals

Edit:
To clarify the statement above. When entering the nether in creative. There isn't an issue. But when trying to go back, it breaks fully. Portals stop working. Once disable Immersive Portals, it works like it should again.

**Comment by qouteall** [comment]
"When entering the nether in creative" Are you using vanilla nether portal?

**Comment by GuldTasken** [comment]
Yup. 2x3 portal. The veeery standard. Also tried a smaller one. Both experienced the same issue.
